compendium/DeclarativeServices/test/gtest/TestBindingPolicies.cpp

```diff
@@ -265,6 +265,14 @@ namespace cppmicroservices
                                                false,
                                                MakeInterfaceMap<test::Interface1>(std::make_shared<test::InterfaceImpl>(
                                                    "ServiceComponentDynamicReluctantMandatoryUnary Interface1")) },
+                            DynamicRefPolicy { "TestBundleDSDRMU_wrongSig",
```

Comment:
Is this test component testing a failure mode? I would think it would from its name (i.e. "wrongSig"). Looking at this test component, I don't see what is "wrong" with it.